UPGRADING.md

```diff
@@ -158,6 +158,111 @@ The `gogoproto.goproto_stringer = false` annotation has been removed from most p
 In this section we describe the changes made in Cosmos SDK' SimApp.
 **These changes are directly applicable to your application wiring.**
 
+#### Unordered Transactions
+
+The Cosmos SDK now supports unordered transactions. This means that transactions
+can be executed in any order and doesn't require the client to deal with or manage
+nonces. This also means the order of execution is not guaranteed.
+
+Unordered transactions are automatically enabled when using `depinject` / app di, simply supply the `servertypes.AppOptions` in `app.go`:
```

Comment:
we should think about whether we want this to be true - maybe we don't want it to be enabled by default

Reply:
is there a way to disable this if you are a depinject user?

Reply:
yes this is true. i will make an issue to explore making this optional.